File: back/model/player.go
package model

import "time"

// Player holds information about a player session
type Player struct {
	PlayerID  string    `json:"playerId"`
	Name      string    `json:"name"`
	Mail      string    `json:"mail"`
	Language  string    `json:"language"`
	StartTime time.Time `json:"startTime"`
}

// PlayerResult is the result of a player
type PlayerResult struct {
	Player
	Score int    `json:"score"`
	Time  string `json:"time"`
}

File: server/src/main/kotlin/com/zenika/nckotlinserver/model/Player.kt
package com.zenika.nckotlinserver.model

import java.time.LocalTime

/**
 * Holds information about a player session
 */
data class Player(
        val playerId: String,
        val name: String,
        val mail: String,
        val language: String,
        val startTime: LocalTime
) : Entity {
    override fun id(): String = playerId
}

/**
 * Represents the result of a player
 */
data class PlayerResult(
        val player: Player,
        val score: Int,
        val time: String
)